src/checkpoint/container/view_serialize.h

```diff
@@ -475,7 +531,7 @@ inline void serialize_const(SerializerT& s, Kokkos::View<T,Args...>& view) {
   using T_non_const = typename ViewType::traits::non_const_data_type;
   Kokkos::View<T_non_const,Args...> tmp_non_const(view.label(), view.layout());
   if (s.isPacking() || s.isSizing()) {
-    Kokkos::deep_copy(tmp_non_const, view);
+    deepCopyWithLocalFence(tmp_non_const, view);
```

Comment:
It's possible that we'd double-copy a device-space view of `const T`. We should see about folding those cases to a single copy, if possible. Perhaps by just aliasing the original const view with a non-const one, and skipping this step.

Reply:
https://github.com/DARMA-tasking/magistrate/issues/245